lib/topics/state_and_behaviour.rb

```diff
@@ -2,4 +2,27 @@
 
 # exercise state and behaviour
 module StateAndBehaviour
+  # define class Car
+  class Car
+    attr_accessor :year, :color, :model, :current_speed
+    def self.default_car
+      Car.new {}
+    end
+
+    def initialize(car_attributes)
+      raise ArgumentError, 'Expects hash' unless car_attributes.is_a?(Hash)
+      self.year = car_attributes.fetch(:year, 2015)
```

Comment:
@sveredyuk at one of the comments above there was told to add getter/setter instead of instance variables